helm/managed/logging/elastic/values.yaml

```diff
@@ -21,6 +28,35 @@ elasticsearch:
       # Add default labels for the volumeClaimTemplate fo the StatefulSet
       #enabled: false
     #annotations: {}
+  extraEnvs:
+    - name: WAIT_ES_READY_IN_SEC
+      value: "100"
+    #- name: STATUS_COLOR
+    #  value: "yellow"
+  lifecycle:
+    postStart:
+      exec:
+        command:
+          - bash
+          - -c
+          - |
+            PIPELINE_NAME=hpccpipeline
+            ES_URL=http://localhost:9200
+            ES_CLUSTER_HEALTH_PATH=_cluster/health
+            ES_CLUSTER_HEALTH_PARAMS=_cluster/health
+            PIPELINE_REQ_PATH=_ingest/pipeline
+            [ -z ${WAIT_ES_READY_IN_SEC} ] && WAIT_ES_READY_IN_SEC=100
+            END_TIME=$(expr $(date +%s) \+ ${WAIT_ES_READY_IN_SEC})
+            ##START_FILE=/tmp/.es_start_file
+            ##ls $START_FILE
+            while [[ "$(curl -s -o /dev/null -w '%{http_code}\n' $ES_URL)" != "200" ]]; do
+              echo "\nES not ready..."
+              [ $(date +%s) -ge ${END_TIME} ] && break
+              sleep 1
+            done
+            #curl -X GET "$ES_URL/$ES_CLUSTER_HEALTH_PATH?wait_for_status=${STATUS_COLOR}&timeout=50s&pretty"  --fail --silent --show-error -o /dev/stderr
+            curl -X PUT "$ES_URL/$PIPELINE_REQ_PATH/$PIPELINE_NAME/" -H 'Content-Type: application/json' \
+              -d'{ "description": "Parses and formats HPCC Systems component log entries", "processors": [{ "grok": { "field": "message", "patterns": ["%{BASE16NUM:hpcc.log.sequence}\\s+%{HPCC_LOG_AUDIENCE:hpcc.log.audience}\\s+%{HPCC_LOG_CLASS:hpcc.log.class}\\s+%{TIMESTAMP_ISO8601:hpcc.log.timestamp}\\s+%{POSINT:hpcc.log.procid}\\s+%{POSINT:hpcc.log.threadid}\\s+%{HPCC_LOG_WUID:hpcc.log.jobid}\\s+%{QUOTEDSTRING:hpcc.log.message}"], "pattern_definitions": { "HPCC_LOG_WUID": "([A-Z][0-9]{8}-[0-9]{6})|(UNK)", "HPCC_LOG_CLASS": "DIS|ERR|WRN|INF|PRO|MET|UNK", "HPCC_LOG_AUDIENCE": "OPR|USR|PRG|AUD|UNK" } } }], "on_failure": [{ "set": { "field": "error.message", "value": "{{ _ingest.on_failure_message }}" } }] }'
```

Comment:
Line 106 needs to be un-commented

Reply:
no, line 106. I mentioned none of this works if line 106 isn't edited. You need to remove the "#" character:
       # pipeline: 'hpccpipeline'
==>
        pipeline: 'hpccpipeline'